bip-taro-proof-file.mediawiki

```diff
@@ -114,7 +114,14 @@ blob with the following format:
 * type: 7 (<code>split_root_proof</code>)
 ** value:
 *** [<code>...*byte</code>:<code>taro_taproot_proof</code>]
-* type: 8 (<code>taro_input_splits</code>)
+* type: 8 (<code>meta_reveal</code>)
+** value:
+*** [<code>...*byte</code>:<code>asset_meta_reveal</code>]
+**** type: 0 (<code>meta_type</code>
```

Comment:
Do we also want to define the single currently defined type (opaque) in this doc?

Reply:
Yeah good idea, will tack that on. 

Based on convos with JHB, I think it might actually be better to make the reveal _mandatory_ for genesis proof files. This way we're able to sidestep issue related to non-cannonical universe root commitments (for each asset a root w/ and w/o optional fields like these).